runtime/syntax/man.vim

```diff
@@ -18,6 +18,11 @@ highlight default link manOptionDesc     Constant
 highlight default link manReference      PreProc
 highlight default link manSubHeading     Function
 
+if &filetype != 'man'
+  " Loaded by the location list TOC
+  finish
```

Comment:
>  It actually makes the syntax loadable by other syntaxes.

Let's just change the comment then. Currently it specifically says "Loaded by the location list TOC" which is confusing.

    " syntax/man.vim may be included by other syntax files.